core/Plugin.php

```diff
@@ -359,7 +360,7 @@ public function findComponent($componentName, $expectedSubclass)
             $this->cache->save($cacheId, $classname);
         }
 
-        return new $classname;
+        return StaticContainer::get($classname);
```

Comment:
`findMultipleComponents()` returns an array of class names, it doesn't instantiate the classes. That is done in other classes, so maybe we can do that later to keep the changes simple.